multiqc/modules/peddy/peddy.py

```diff
@@ -74,10 +74,11 @@ def __init__(self):
 
         # Relatedness plot
         self.peddy_relatedness_plot()
-
+        
```

Comment:
Would be nice if you could clean up these whitespace changes in the diff - they all get highlighted in a bright orange in my editor so it saves me removing them later 😉